README.md

```diff
@@ -1 +1,66 @@
-# loaderboard-proj
\ No newline at end of file
+# loaderboard-proj
+In this project I built a Leaderboard list app, where the following was implemented in the project
+- Created a new game using the API.
+- saved the ID of my game that will be returned by API.
+- Implemented the "Refresh" button that gets all scores for the game created from the API.
+- Implemented the form "Submit" button that will save scores for the game.
+- Used arrow functions instead of the function keyword.
+- Used async and await JavaScript features to consume the API.
+## Built With
+
+- HTML
+- CSS
+-javascript
```

Comment:
- [OPTIONAL] I think it's better to put a blank space between the `-` and the `javascript`. Will make your README looks more professional. For more information [click here](https://www.markdownguide.org/cheat-sheet/). 
- [OPTIONAL] Please do the same with the `-webpack` on line 15.